pkg/controller/operators/catalog/operator.go

```diff
@@ -1627,6 +1637,14 @@ func (o *Operator) syncInstallPlans(obj interface{}) (syncError error) {
 	if len(plan.Status.BundleLookups) > 0 {
 		unpacked, out, err := o.unpackBundles(plan)
 		if err != nil {
+			// If the error was fatal capture and fail
+			if retryable := olmerrors.IsFatal(err); !retryable {
```

Comment:
same here, I think we want
```
if fatal := olmerrors.IsFatal(err); fatal {}
```

Reply:
Oof, absolutely, yeah. Completely my bad. I just searched and replaced for instances of the actual error. Somehow didn't think about references pointing to it. Friday brain.